Dockerfile

```diff
@@ -9,7 +9,8 @@ WORKDIR /prod
 
 # First, pip install dependencies
 COPY requirements.txt requirements.txt
-RUN pip install -r requirements.txt
+COPY processed_data/smaller_data_sample_text.csv smaller_data_sample_text.csv
```

Comment:
do we need both?

Reply:
there is just one version that's copied from the processed_data folder to the docker image as smaller_data_sample_text.csv